src/platform/cocoamain.mm

```diff
@@ -1218,6 +1218,227 @@ - (IBAction)preferences:(id)sender {
     [NSApp stop:nil];
 }
 
+#ifdef HAVE_SPACEWARE
```

Comment:
Oh, now that you mention it, this is no longer needed. Whoopsie, I'll remove it...

But I'm not linking to anything anymore.